frontend/src/app/modules/matches/matches-saga.test.js

```diff
@@ -0,0 +1,55 @@
+import { addMatch } from './matches-effects'
+import { addMatchSaga } from './matches-saga'
+import { call, put, select } from 'redux-saga/effects'
+import { MatchesActions } from './matches-actions'
+import { inProgress, success, failure } from '../api/request-status'
+import { getUsersSaga } from '../users/users-saga'
+import { getSelectedGame } from '../games/games-selectors'
+
+const MATCH = {}
+const FOOSBALL = { name: 'foosball' }
+const ERROR = {
+  message: 'ERROR',
+}
+
+describe('matchesSaga', () => {
+  describe('addMatchSaga', () => {
+    let gen
+    const expectNextToBe = (value, done = false) => {
```

Comment:
nice little helper ;) As a idea: since the unit tests here serve strictly as a regression. I'd consider snapshot test this, to save the hassle. However, this is fine as well!

Reply:
agreed to leave as it is